testsuite/features/upload_files/health_check_setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+HEALTH_CHECK_PACKAGE="python311-mgr-health-check"
+HEALTH_CHECK_REPO="https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/healthcheck/15.6/"
+HEALTH_CHECK_CONFIG="/usr/lib/python3.11/site-packages/health_check/config.toml"
```

Comment:
No please, all this should be in sumaform, let's provision the tool during the deployment through sumaform.
We don't want to handle repository definitions of our test infrastructure as part of the test framework.